### ISSUE TITLE
Docs: Add a link to Open JDK on getting started page.

### ISSUE THREAD

**Issue by toshihidetagami** [playframework/playframework#10938]
## Problem

Currently, the documentation suggests installing OracleJDK if you don’t have java installed.
The problem is OracleJDK 11 or higher is commercial and not free. ([reference](https://blog.joda.org/2018/09/do-not-fall-into-oracles-java-11-trap.html))
So I can imagine there will be some beginners who visit on getting started page install the commercial version without realizing it and get into trouble.

**Pages OracleJDK is mentioned**
[Getting Started with Play Framework](https://www.playframework.com/getting-started)
[Play Requirements](https://www.playframework.com/documentation/2.8.x/Requirements)

## Proposal
Add a link to the OpenJDK.
[Scala’s getting started page](https://docs.scala-lang.org/getting-started/index.html) suggests AdoptOpenJDK so we can follow it. 
Or just a link to [JDK compatibility in scala's documentation ](https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html)might be better.

**Comment by ignasi35** [comment]
thanks @toshihidetagami for raising this. I could find a reference in https://www.playframework.com/getting-started.

Do you have other references on the play's site in mind?

**Comment by ignasi35** [comment]
I found a few other references searching for "Oracle’s JDK Site" under https://github.com/playframework/playframework/blob/master/documentation/

Note the playframework website is the result of combining sources from several repositories. This is why I also raised https://github.com/playframework/playframework.com/issues/445.

**Comment by ignasi35** [comment]
@toshihidetagami would you be able to work on a couple of PRs?

**Comment by toshihidetagami** [comment]
@ignasi35 Yes, I would like to work on this. I have never worked on this project, so I will take some time to read the code in this repository and playframework/playframework.com first.
I will check if other pages have oracle's link, but I think the most important one is getting started page.

Regarding change, will we just replace oracle's link to AdoptOpenJDK 's link? Or any other website in your mind?
(I checked adoptium as mentioned in playframework/playframework.com#445, it is almost identical to AdoptOpenJDK's website but I found some pages are still work in progress)
